input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,506 @@
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-smokingstatus"
-                    >
-                        US Core Smoking Status Observation
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
-                </p>
-            </td>
-        </tr>
-        <tr>
-            <td width="93" valign="top">
-                <p>
-                    <a
-                        href="StructureDefinition-au-core-waistcircum.html"
-                    >
-                        AU Core Waist Circumference
-                    </a>
-                </p>
-            </td>
-            <td width="81" valign="top">
-                <p>
-                    <a
-                        href="https://build.fhir.org/ig/HL7/fhir-ipa/StructureDefinition-ipa-observation.html"
-                    >
-                        IPA-Observation
-                    </a>
-                </p>
-            </td>
-            <td width="111" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
-                </p>
-            </td>
-            <td width="68" valign="top">
-            </td>
-            <td width="130" valign="top">
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-simple-observation"
-                    >
-                        US Core Simple Observation
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
-                </p>
-            </td>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-condition-problems-health-concerns.html">US Core Condition Problems and Health Concerns</a></td>
+            <td style="width: 25%;">Category.code</td>
+            <td style="width: 25%;">US Core requires a value from <a href="https://hl7.org/fhir/us/core/STU7/ValueSet-us-core-problem-or-health-concern.html">US Core Problem or Health Concern</a> value set.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-condition-encounter-diagnosis.html">US Core Condition Encounter Diagnosis Profile</a></td>
+            <td style="width: 25%;">Category.code</td>
+            <td style="width: 25%;">US Core requires 'encounter-diagnosis' value.</td>
```

Comment:
Missing entry for Condition.category to state for the [US Core Condition Problems and Health Concerns Profile](https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-condition-problems-health-concerns.html).

State "US Core requires category of 'problem-list-item' or 'health-concern'."

Reply:
Done